### PR TITLE
fix minor typos in CoC

### PR DESCRIPTION
**Pull request by vchrombie**
Signed-off-by: Venu Vardhan Reddy Tekula <venuvardhanreddytekula8@gmail.com>

### CODE REVIEW

**Comment by vchrombie** [comment]
Once this PR approved and merged, I would like to send a PR to wg-d&i for the changes in CoC and wg-gmd for adding the CoC.

File: code-of-conduct.md [hunk 1]
```diff
@@ -74,8 +74,7 @@ CHAOSS members or the general public. We will not name victims without their
 affirmative consent.
 
 The CHAOSS Code of Conduct Team will report on a yearly basis the number of
-incidence reports. The CHAOSS Code of Conduct Team shall consist of three members 
-elected by the community every two years or when needed.
+incidence reports. The CHAOSS Code of Conduct Team shall consist of three members elected by the community every two years or when needed.
```
Comment (jgbarah) [inline code-of-conduct.md]: This is better in two lines, as it was before this patch (that helps to readability of the unformatted text.

File: code-of-conduct.md [hunk 2]
```diff
@@ -48,13 +48,13 @@ threatening, offensive, or harmful.
...
 representing a project or community include using an official project e-mail
-address, posting via an official social media account, or acting as an appointed
+address, posting via an official social media account or acting as an appointed
 representative at an online or offline event. Representation of a project may be
 further defined and clarified by project maintainers.
 
 ## Enforcement
 
-Instances of abusive, harassing, or otherwise unacceptable behavior may be
+Instances of abusive, harassing, or otherwise, unacceptable behavior may be
```
Comment (germonprez) [inline code-of-conduct.md]: No comma needed here.

**Comment by GeorgLink** [comment]
> Once this PR approved and merged, I would like to send a PR to wg-d&i for the changes in CoC and wg-gmd for adding the CoC.

That is an excellent idea!

Can you please make the requested changes by committing to the same branch this pull request was created from?

File: code-of-conduct.md (hunk 1 above)
Comment (vchrombie) [reply code-of-conduct.md]: Ok. I will revert this change back.

File: code-of-conduct.md (hunk 2 above)
Comment (vchrombie) [reply code-of-conduct.md]: Ok, I will revert the change.

**Comment by vchrombie** [comment]
I have pushed the required changes. You can review it once whenever you are free. :)

**Comment by vchrombie** [comment]
Thanks, @jgbarah. :)

**Pull request merged**